x/ibc/03-connection/client/cli/tx.go

```diff
@@ -371,7 +369,6 @@ func GetCmdHandshakeState(storeKey string, cdc *codec.Codec) *cobra.Command {
 			// Create and send msgUpdateClient
 			viper.Set(flags.FlagChainID, cid2)
 			msgUpdateClient := ibcclient.NewMsgUpdateClient(clientID2, header, ctx2.GetFromAddress())
-			fmt.Printf("%v <- %-23v", cid2, msgUpdateClient.Type())
```

Comment:
I would like to leave this logging in as a way for users to understand what is happening in the transaction handshake. Can we please revert these changes??